test/src/e2e_vm_tests/mod.rs

```diff
@@ -109,7 +109,12 @@ impl TestContext {
 
                 let (result, ..) = harness::compile_to_bytes(&name, &context.run_config, false);
                 assert_matches!(result, Ok(_));
-                let compiled = result.unwrap();
+                let compiled = match result.unwrap() {
+                    forc_pkg::Built::Package(built_pkg) => *built_pkg,
+                    forc_pkg::Built::Workspace(_) => {
+                        panic!("workspaces are not supported in the test suite yet")
```

Comment:
I am looking into enabling workspace support in the test suite, it seems like I might need to play with the structure a little but I guess it would be cool to support them with this PR. What are your thoughts? cc @mitchmindtree

Reply:
Testing suite can now have "compile" tests for workspaces but not run. I am planning to add it with `forc-client` workspace support pr.